### PR TITLE
Branch ugdg update

### PR DESCRIPTION
**Pull request by raydenlim**
Updated UserGuide.md
Updated DeveloperGuide.md
Updated images for FAKEJARVIS

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -6,11 +6,13 @@ pageNav: 3
...
 
-<img src="images/fakejarvis.png" width="200px">
+<p align="center">
+  <img src="images/fakejarvis.png" width="200px">
+</p>
 
 _Say goodbye to chaos and hello to an organized and efficient classroom experience._
 
-F.A.K.E.J.A.R.V.I.S. is the **ultimate solution for CS1101S Avengers who manage a multitude of tutoring tasks.** This platform empowers you to efficiently track assignment gradings, monitor student participation, and seamlessly plan tutorials, consultations, and mastery checks. This is your one-stop solution to managing your time and students as a CS1101S Avenger!
+F.A.K.E.J.A.R.V.I.S. is the **ultimate solution for CS1101S Avengers who manage a multitude of tutoring tasks.** This platform empowers you to efficiently track assignment gradings, monitor student participatsion, and seamlessly plan tutorials, consultations, and mastery checks. This is your one-stop solution to managing your time and students as a CS1101S Avenger!
```
Comment (Respirayson) [inline docs/UserGuide.md]: perhaps this is a typo

File: docs/UserGuide.md [hunk 2]
```diff
@@ -85,8 +87,6 @@ Adds a person to the address book.
 
 Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
 
-<box type="tip" seamless>
```
Comment (Respirayson) [inline docs/UserGuide.md:88]: I think the box is for styling? Not sure whether can just delete

**Review by Respirayson — COMMENTED** [review]
just a few nits otherwise looks good

File: docs/UserGuide.md [hunk 3]
```diff
@@ -250,8 +248,64 @@ Examples:
...
+### 🔎Finding a Student: find
+Finds a student’s profile
+
+Format: `find n/STUDENT_NAME`
+
+Parameters:
+* STUDENT_NAME - The name of the student you want to find.
+
+Examples:
+* `find n/John Doe` Finds and displays the profile of students with the name “John Doe.”
```
Comment (ldinghan) [inline docs/UserGuide.md]: Perhaps full stop should be outside quotation marks, i.e. '"John Doe".'

File: docs/UserGuide.md [hunk 4]
```diff
@@ -250,8 +248,64 @@ Examples:
...
+Finds a student’s profile
+
+Format: `find n/STUDENT_NAME`
+
+Parameters:
+* STUDENT_NAME - The name of the student you want to find.
+
+Examples:
+* `find n/John Doe` Finds and displays the profile of students with the name “John Doe.” 
+* `find n/Jane Doe` Finds and displays the profile of students with the name “Jane Doe.”
```
Comment (ldinghan) [inline docs/UserGuide.md]: same as above comment

File: docs/UserGuide.md (hunk 2 above)
Comment (raydenlim) [reply docs/UserGuide.md:88]: oops mb, will add it back

**Review by ldinghan — APPROVED** [review]
LGTM

**Pull request merged**